guides/common/modules/proc_creating-compute-profiles.adoc

```diff
@@ -19,3 +22,44 @@ You can apply compute profiles to all supported compute resources:
 . Click *Submit*.
 A new window opens with the name of the compute profile.
 . In the new window, click the name of each compute resource and edit the attributes you want to set for this compute profile.
+
+[id="cli-creating-compute-profiles_{context}"]
+.CLI procedure
+. Create compute profile:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile create --name "_My_Compute_Profile_"
+----
++
+. Set attributes for compute profile:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile values create \
+--compute-attributes "flavor=m1.small,cpus=2,memory=4GB,cpu_mode=default \
+--compute-resource "_My_Compute_Resource_" \
+--compute-profile "_My_Compute_Profile_" \
+--volume size=40GB
+----
++
+. Optional: To update the attributes of a compute profile, specify the attributes you want to change. To change the number of CPUs, memory, volume size use the following command:
```

Comment:
```suggestion
. Optional: To update the attributes of a compute profile, specify the attributes you want to change.
For example, to change the number of CPUs and memory size:
```
This is mostly just about the conventions for the Foreman project (for example, we use the 'one sentence per line convention', so I added a new line).

Reply:
Yes i did, thanks for mentioning it